cucumber/api/features/step_definitions/request_steps.rb

```diff
@@ -103,6 +103,31 @@
   end
 end
 
+When(/^I authenticate Alice with Accept-Encoding header "([^"]*)"(?: with plain text body "([^"]*)")?$/) do |header, body|
+  body ||= ":cucumber:user:alice_api_key"
+  authn_request(
+    url: "/authn/cucumber/alice/authenticate",
+    api_key: body,
+    encoding: header.downcase.include?("base64"),
+    can: false
+  )
+end
+
+When(/^I authenticate Alice "([^"]*)" times in "([^"]*)" threads with Accept-Encoding: base64 header?$/) do |requests_num, threads_num|
+  body ||= ":cucumber:user:alice_api_key"
+  authenticate_with_performance(
+    requests_num,
+    threads_num,
+    authentication_func: :authn_request,
+    authentication_func_params: {
+      url: "/authn/cucumber/alice/authenticate",
+      api_key: body,
+      encoding: true,
+      can: true
```

Comment:
what is the usage of `can`? i don't understand it.

Reply:
`try_request` uses `can` boolean parameter to  whether to raise exception if the http call failed or keep the test going (ended with error code)